king_phisher/client/windows/plugin_manager.py

```diff
@@ -400,7 +391,7 @@ def _plugin_install_tsafe(self, catalog_model, repo_model, model_row, named_row)
 				self._update_status_bar_tsafe(
 					"Installing {:,} dependenc{} for plugin {} from PyPi.".format(len(packages), 'y' if len(packages) == 1 else 'ies', named_row.title)
 				)
-				pip_results = self._pip_install(packages)
+				pip_results = self.application.plugin_manager.install_packages(packages)
```

Comment:
You should either catch the exception that `install_packages` might raise or check that `plugin_manager.library_path` is set before calling this.